src/Profiles/LEV/DataPages/RX/ANTPLUS_LevSpeedDist.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Profiles/Lev/DataPages/RX/ANTPLUS_LevSpeedDist.h>
+#include <Profiles/Lev/ANTPLUS_LevPrivateDefines.h>
+
+/* Speed and Distance */
+LevSpeedDist1::LevSpeedDist1(AntRxDataResponse& dp) : LevBaseMainDataPage(dp) {
+	return;
+}
+
+uint32_t LevSpeedDist1::getTotalDist() // in km
```

Comment:
Can we call this Odometer? Just so it matches the datasheet and its easier to look up?